### DISCUSSION TITLE
Anyone able to get Mantine UI working with Deno/Fresh

### DISCUSSION THREAD

**Discussion by OldManFroggy** [mantinedev/mantine#1961] [Q&A]
Subject line says it all, I have tried using the esm.sh shims but am not having much luck so far.

Interested to see if anyone has got this working, if not -> a pointer into where issue lies would help me greatly (looks, like a preact issue -> which I am not as experienced with as I would like) .

**Comment by chrisdevelops** [comment]
Did you ever get Mantine and Fresh working together? I am wanting to use the two together myself.
